redaxo/src/addons/structure/plugins/content/boot.php

```diff
@@ -93,3 +93,15 @@
         rex_response::sendPage($content, $article->getValue('updatedate'));
     });
 }
+
+rex_extension::register('EDITOR_URL', function (rex_extension_point $ep) {
+    static $urls = [
+        'template' => ['templates', 'template_id'],
+        'module' => ['modules/modules', 'module_id'],
+        'action' => ['modules/actions', 'action_id'],
+    ];
+
+    if (preg_match('@^rex:///(template|module|action)/(\d+)@', $ep->getParam('file'), $match)) {
+        return rex_url::backendPage($urls[$match[1]][0], ['function' => 'edit', $urls[$match[1]][1] => $match[2]]);
```

Comment:
ich weiß nicht ob code-mirror das kann, aber ggf. ist es ja möglich in den textareas auch zur "korrekten" zeile zu springen via codemirror?

Reply:
Vor allem müsste dann aber auch zur richtigen Textarea gesprungen werden (zum Beispiel Eingabe vs Ausgabe bei Modulen).